internal/constant/cmd.go

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023. Nydus Developers. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// constants of nydus snapshotter CLI config
+
+package constant
+
+const (
```

Comment:
Why the source file is named as cmd.go ?

Reply:
I think the constants in this file are prepared for the cmd and config package, what name do you think is better?

Reply:
how about values.go or constants.go?

Reply:
values.go looks good.